qiskit_finance/data_providers/random_data_provider.py

```diff
@@ -35,7 +35,7 @@ def __init__(
         Initialize RandomDataProvider.
 
         Args:
-            tickers (str | list[str] | None): Tickers for the data provider.
+            tickers (list[str] | None): Tickers for the data provider.
```

Comment:
Given lins 49 thru 53 do instance list else where in the else it splits a str I think this Args and the type in the init should be list[str] | str | None in this and the other class. As it was before in the init it was ` tickers: Optional[Union[str, List[str]]] = None,` so its lost the str type in the conversion to | along the way and now this has too.

Reply:
Is the `str` type there to cover this functionality?
```python
self._tickers = tickers.replace("\n", ";").split(";")
```

Reply:
Yes, the else part of the instance list test.